examples/step-35/step-35.cc

```diff
@@ -796,11 +796,11 @@ namespace Step35
         vel_exact.set_time(t_0);
         vel_exact.set_component(d);
         VectorTools::interpolate(dof_handler_velocity,
-                                 Functions::ZeroFunction<dim>(),
+                                 vel_exact,
                                  u_n_minus_1[d]);
         vel_exact.advance_time(dt);
         VectorTools::interpolate(dof_handler_velocity,
-                                 Functions::ZeroFunction<dim>(),
+                                 vel_exact,
                                  u_n[d]);
```

Comment:
The continuous integration tests complain that you should move the arguments for this second call all to the same line. Have you used the automatic indenting of deal.II before? If you're interested, you can look that up here:
https://www.dealii.org/developer/doxygen/deal.II/CodingConventions.html

Reply:
Thank you. I formated the code and updated the PR.